docs/release-promotion/desktop/howto_52esr_only.md

```diff
@@ -85,6 +80,52 @@ Examples
 
 ### How
 
+* **If Plan A happened for pushing to mirrors and graph expired**, you'll need to create it according sans the pushing-to-mirrors (which at this point already happened).
+
+```bash
+ssh `whoami`@buildbot-master85.bb.releng.scl3.mozilla.com  # host we release-runner and you generate/submit new release promotion graphs
+sudo su - cltbld
+TASK_TASKID_FROM_GRAPH1={insert a taskid from any task in graph 1}
+BRANCH_CONFIG=prod_mozilla-esr52_firefox_rc_graph_2.yml
+
+cd /home/cltbld/releasetasks/
+git pull origin master  # make sure we are up to date. note: make sure this is on master and clean first
+vi releasetasks/release_configs/prod_mozilla-esr52_firefox_rc_graph_2.yml # remove push-to-mirrors task from second graph
+git diff # you should have something similar to this
++++ b/releasetasks/release_configs/prod_mozilla-esr52_firefox_rc_graph_2.yml
+@@ -29,7 +29,7 @@ updates_enabled: false
+ checksums_enabled: false
+ push_to_candidates_enabled: false
+ update_verify_enabled: false
+-push_to_releases_enabled: true
++push_to_releases_enabled: false
+ channels:
+     - "esr"
+ publish_to_balrog_channels:
+@@ -51,7 +51,7 @@ funsize_balrog_api_root: "http://balrog/api"
+ balrog_api_root: "https://aus4-admin.mozilla.org/api"
+ build_tools_repo_path: "build/tools"
+ tuxedo_server_url: "https://bounceradmin.mozilla.com/api"
+-push_to_releases_automatic: true
++push_to_releases_automatic: false
+ beetmover_candidates_bucket: "net-mozaws-prod-delivery-firefox"
+ snap_enabled: false
+ update_verify_channel: null
+
+cd /builds/releaserunner/tools/buildfarm/release/
+hg pull -u # make sure we are up to date. note: make sure this is on default and clean first
+source /builds/releaserunner/bin/activate
+
+# call releasetasks_graph_gen.py with --dry-run and sanity check the graph output that would be submitted
+python releasetasks_graph_gen.py --release-runner-config=../../../release-runner.yml --branch-and-product-config="/home/cltbld/releasetasks/releasetasks/release_configs/${BRANCH_CONFIG}" --common-task-id=$TASK_TASKID_FROM_GRAPH1 --dry-run
+
+# call releasetasks_graph_gen.py for reals which will submit the graph to Taskcluster
+python releasetasks_graph_gen.py --release-runner-config=../../../release-runner.yml --branch-and-product-config="/home/cltbld/releasetasks/releasetasks/release_configs/${BRANCH_CONFIG}" --common-task-id=$TASK_TASKID_FROM_GRAPH1
+
+```
+* **If Plan A happened for pushing to mirrors and graph hasn't expired**, proceed with the below instructions
```

Comment:
Perhaps:

**If Plan A from above "Push artifacts to releases directory" and graph hasn't expired**, proceed with the below instructions against the original, single graph for esr52.